### DISCUSSION TITLE
Demo

### DISCUSSION THREAD

**Discussion by btessler92** [small-cactus/M.I.L.E.S#5] [Show and tell]
Hi Can you show MILES working? I want to see the speed and how it looks. Maybe just ask it organize your chrome tabs and add new tabs of searching something on google

**Comment by small-cactus** [comment]
Miles can not control chrome, Miles can only gather search results. Here is a Demo. (after new wake word model update)
Play with audio!!

https://github.com/small-cactus/M.I.L.E.S/assets/125771841/637b180d-f644-47d8-8ee2-d35e1f0487ba
